sim/side.ts

```diff
@@ -184,6 +184,12 @@ export class Side {
 					if (move === 'hiddenpower') {
 						return move + toID(pokemon.hpType) + (pokemon.hpPower === 70 ? '' : pokemon.hpPower);
 					}
+					if (move === 'frustration' || move === 'return') {
```

Comment:
Possibly only want to include when not 102, but the code in `sim/pokemon.ts` includes it always so I figured I might as well match

Reply:
It makes sense to include when it _is_ 102. "Known to be 102" and "Unknown base power" would otherwise be ambiguous.